### DISCUSSION TITLE
Alguém pode me explicar o que eu fiz de errado??

### DISCUSSION THREAD

**Discussion by AngryPlayer04** [frontendbr/forum#1678] [Perguntas e Respostas]
Estou aprendendo JS e fiz esse teste, mas a soma só da número inteiro, e eu quero que dê número com vírgula, não sei o que fazer.`

[ex003.txt](https://github.com/frontendbr/forum/files/4125437/ex003.txt)

**Comment by theandersonn** [comment]
@AngryPlayer04 blz?

Por padrão o separador de casas decimais no float é o ponto [.] seu código inclusive já funciona passando valores separados por ponto... para aceitar virgulas, é preciso depois que a variavel receber os respectivos valores tratar, pode fazer isso com replace, exemplo:

```
  var n1 = window.prompt('Digite um número: ')
  var n2 = window.prompt('Digite outro número: ')
  n1 = parseFloat(n1.replace(',','.'))
  n2 = parseFloat(n2.replace(',','.'))
  var S = n1 + n2
  window.alert('A soma dos valores é ' + S )
```

Teste aí...

**Comment by carlosviniciusananias** [comment]
Oi AngryPlayer04, 

Você pode dar também utilizar o 'toLocaleString' para isso. Ele funciona bem em navegadores modernos e você não vai precisar esquentar cabeça com casas decimais e concatenação o cifrão.

Ex;
toLocaleString('pt-BR', {
        minimumFractionDigits: 2,
        maximumFractionDigits: 2
      });

**Comment by felipefialho** [comment]
@AngryPlayer04 Por favor, troca o titulo para descrever melhor qual a dúvida? Obrigado :)
